src/js/components/charts/HostTimeSeriesChart.js

```diff
@@ -27,10 +26,10 @@ var HostTimeSeriesChart = React.createClass({
   getMaxY: function () {
     var props = this.props;
     var roundUpValue = props.roundUpValue;
-
-    var maxSlavesCount = _.max(props.data, function (slave) {
-      return slave.slavesCount;
-    }).slavesCount;
+    let slavesCounts = props.data.map(function (agent) {
```

Comment:
This could be written as a reduce:
```js
let maxSlavesCount = props.data.reduce(function (max, {slavesCount}) {
  if (max < slavesCount) {
    return slavesCount;
  }

  return max;
}, 0);
```

But not sure which one is prettier `¯\_(ツ)_/¯`